src/frontend/apps/desk/src/features/language/LanguagePicker.tsx

```diff
@@ -60,14 +60,29 @@ export const LanguagePicker = () => {
     }));
   }, [languages]);
 
+  useEffect(() => {
+    if (!document) {
+      return;
+    }
+    const languagePickerDom = document.querySelector(
+      '.c__select-language-picker',
+    );
+
+    if (!languagePickerDom?.firstChild?.firstChild) {
+      return;
+    }
+
+    (languagePickerDom.firstChild.firstChild as HTMLElement).tabIndex = -1;
```

Comment:
```suggestion
    document
      .querySelector('.c__select-language-picker .c__select__wrapper')
      ?.setAttribute('tabindex', '-1');
```

Reply:
Can you add a small comment above the useEffect to explain what it is doing ?

Reply:
I think you can simplify, no?

```suggestion
     document  
      .querySelector('.c__select-language-picker .c__select__wrapper')  
      ?.setAttribute('tabindex', '-1'); 
```